modules/cnf-topology-aware-lifecycle-manager-backup-recovery.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+// Epic CNF-3901 (CNF-2133) (4.11), Story TELCODOCS-339
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="talo-backup-recovery_{context}"]
+= Recovering after a failed upgrade
+
+If an upgrade fails, you can use the backup to return the system to its pre-upgrade state. There are two stages:
+
+Rollback:: If the attempted upgrade included a change to the platform OS deployment, it will be necessary to rollback to the previous version before running the recovery script.
+Recovery:: The recovery shuts down containers, and restores files from the backup partition to the targeted directories. A reboot is necessary. The second stage restores cluster database and related files from the backup, relaunches containers, and triggers redeployments to restore clusters.
```

Comment:
Do you think if we can make it more generic, something like below?
`Recovery:: The recovery shuts down containers, and restores files from the backup partition to the targeted directories and prompts for a reboot. The second stage triggers redeployments to restore clusters.`
If you think it makes more sense then feel free to rephrase.

Reply:
Agreed and changed